### PR TITLE
Change `standard library` term to `Ballerina library`

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Change `standard library` term to `Ballerina library`.
> Fixes #

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: components/ai/code/ai-bbe/create-images-with-dall-e.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ---
 title: 'Create images with DALL-E and Stable Diffusion'
-description: "Stable Diffusion and OpenAI’s DALL-E image APIs generate or edit images with text based instructions. Ballerina’s powerful standard library makes uploading, downloading and processing images a breeze."
+description: "Stable Diffusion and OpenAI’s DALL-E image APIs generate or edit images with text based instructions. Ballerina’s powerful library makes uploading, downloading and processing images a breeze."
```
Comment (anupama-pathirage) [inline components/ai/code/ai-bbe/create-images-with-dall-e.md:3]:
```suggestion
description: "Stable Diffusion and OpenAI’s DALL-E image APIs generate or edit images with text based instructions. The powerful Ballerina libray makes uploading, downloading and processing images a breeze."
```

Comment (anupama-pathirage) [reply components/ai/code/ai-bbe/create-images-with-dall-e.md:3]: IMO Ballerina library going together is better as it is consistent and give correct context. Check and fix all places.

File: components/ai/code/ai-bbe/transcribe-speech-or-music-with-whisper.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
 ---
 title: 'Transcribe speech or music with Whisper'
-description: "OpenAI’s Whisper API makes speech and music computable! Easily transcribe speech or music in any language into text. Ballerina’s powerful standard library makes manipulating audio files and processing the results trivially simple."
+description: "OpenAI’s Whisper API makes speech and music computable! Easily transcribe speech or music in any language into text. Ballerina’s powerful library makes manipulating audio files and processing the results trivially simple."
```
Comment (anupama-pathirage) [inline components/ai/code/ai-bbe/transcribe-speech-or-music-with-whisper.md:3]: ditto

File: components/ai/code/ai-bbe/write-robust-api-powered-ai-applications.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Write robust API-powered AI applications'
-description: "Take on the unpredictable world of distributed systems with Ballerina's built-in language and standard library features. Writing robust API-powered AI applications is now a breeze for every developer, no matter the challenges!"
+description: "Take on the unpredictable world of distributed systems with Ballerina's built-in language and library features. Writing robust API-powered AI applications is now a breeze for every developer, no matter the challenges!"
```
Comment (anupama-pathirage) [inline components/ai/code/ai-bbe/write-robust-api-powered-ai-applications.md:3]: Here in the original text `built-in language and standard library` means : langlib and standard lib in previous terminology. Now those two are combined to one thing named Ballerina library. So it is incorrect to say `built-in language ...` in new text.

File: components/integration/code/integration-bbe/resilience-engineering-built-in.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Resilience engineering, built-in'
-description: Ballerina’s built-in language and standard library features make programming in the failure-is-normal world of distributed systems as easy for every developer.
+description: Ballerina’s built-in language and library features make programming in the failure-is-normal world of distributed systems as easy for every developer.
```
Comment (anupama-pathirage) [inline components/integration/code/integration-bbe/resilience-engineering-built-in.md:3]: ditto

File: components/integration/code/integration-bbe/unleash-the-power-of-ai.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Unleash the power of AI'
-description: "With Ballerina's AI connectors, AI libraries, and built-in language and standard library features, incorporating AI into your integrations can be done in a matter of seconds, enabling you to quickly build intelligent and efficient workflows."
+description: "With Ballerina's AI connectors, AI libraries, and built-in language and Ballerina library features, incorporating AI into your integrations can be done in a matter of seconds, enabling you to quickly build intelligent and efficient workflows."
```
Comment (anupama-pathirage) [inline components/integration/code/integration-bbe/unleash-the-power-of-ai.md:3]: This need full rewrite as AI connectors, AI libraries (incorrect to say libraries here and should use packages) and same comment related to built-in language above applies here too.

File: swan-lake/development-tutorials/build-a-graalvm-executable/build-the-executable-locally.md [hunk 6]
```diff
@@ -199,7 +199,7 @@ Now, you tested a simple Ballerina HTTP server application for GraalVM compatibi
 
 ## Create GraalVM-compatible library packages
 
-Packages that solely use Ballerina standard libraries and connectors or those that use platform Java libraries provided by the distribution are inferred to be compatible with GraalVM.
+Packages that solely use Ballerina library packages and connectors or those that use platform Java libraries provided by the distribution are inferred to be compatible with GraalVM.
```
Comment (anupama-pathirage) [inline swan-lake/development-tutorials/build-a-graalvm-executable/build-the-executable-locally.md:202]: Connectors (i.e. ballerinax packages) are now part of Ballerina library.

File: components/ai/code/ai-bbe/create-images-with-dall-e.md (hunk 1 above)
Comment (praneesha) [reply components/ai/code/ai-bbe/create-images-with-dall-e.md]: Fixed.

File: components/ai/code/ai-bbe/transcribe-speech-or-music-with-whisper.md (hunk 2 above)
Comment (praneesha) [reply components/ai/code/ai-bbe/transcribe-speech-or-music-with-whisper.md:3]: Fixed.

File: components/ai/code/ai-bbe/write-robust-api-powered-ai-applications.md (hunk 3 above)
Comment (praneesha) [reply components/ai/code/ai-bbe/write-robust-api-powered-ai-applications.md:3]: Changed to 'with the power of the Ballerina features` as it is not required to highlight the library features only but all features in general.

File: components/integration/code/integration-bbe/resilience-engineering-built-in.md (hunk 4 above)
Comment (praneesha) [reply components/integration/code/integration-bbe/resilience-engineering-built-in.md:3]: Changed to: `language and library features of Ballerina`

File: components/integration/code/integration-bbe/unleash-the-power-of-ai.md (hunk 5 above)
Comment (praneesha) [reply components/integration/code/integration-bbe/unleash-the-power-of-ai.md:3]: Changed to `language and library features of Ballerina`.

File: swan-lake/development-tutorials/build-a-graalvm-executable/build-the-executable-locally.md (hunk 6 above)
Comment (praneesha) [reply swan-lake/development-tutorials/build-a-graalvm-executable/build-the-executable-locally.md:202]: Changed to `Packages that solely use Ballerina library packages`.

File: components/ai/code/ai-bbe/create-images-with-dall-e.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
 ---
 title: 'Create images with DALL-E and Stable Diffusion'
-description: "Stable Diffusion and OpenAI’s DALL-E image APIs generate or edit images with text based instructions. Ballerina’s powerful standard library makes uploading, downloading and processing images a breeze."
+description: "Stable Diffusion and OpenAI’s DALL-E image APIs generate or edit images with text based instructions. The powerful Ballerina libray makes uploading, downloading and processing images a breeze."
```
Comment (NipunaRanasinghe) [inline components/ai/code/ai-bbe/create-images-with-dall-e.md:3]: WDYT?
```suggestion
description: "Stable Diffusion and OpenAI’s DALL-E image APIs generate or edit images with text based instructions. The power of Ballerina libraries makes uploading, downloading and processing images a breeze."
```

File: swan-lake/development-tutorials/build-a-graalvm-executable/graalvm-executable-overview.md [hunk 8]
```diff
@@ -36,7 +36,7 @@ The only downside is that the GraalVM native image build is a highly complicated
 
 From Ballerina 2201.7.0 (SwanLake) onwards, Ballerina supports GraalVM AOT compilation to generate standalone executables by passing the `graalvm` flag in the build command: `bal build --graalvm`. The generated executable contains the modules in the current package, their dependencies, Ballerina runtime, and statically linked native code from the JDK.
 
-Ballerina runtime, [standard libraries](/learn/ballerina-specifications/#standard-library-specifications), and the Ballerina extended modules are GraalVM-compatible. Therefore packages developed only using these libraries are also GraalVM-compatible. Furthermore, Ballerina reports warnings when the GraalVM build is executed for a project with GraalVM-incompatible packages.
+The Ballerina runtime and [Ballerina library](https://central.ballerina.io/search?q=org%3Aballerina%2Cballerinax&page=1) are GraalVM-compatible. Therefore packages developed only using these libraries are also GraalVM-compatible. Furthermore, Ballerina reports warnings when the GraalVM build is executed for a project with GraalVM-incompatible packages.
```
Comment (NipunaRanasinghe) [inline swan-lake/development-tutorials/build-a-graalvm-executable/graalvm-executable-overview.md:39]: isn't the `libraries` is the correct term to be used here?
```suggestion
The Ballerina runtime and [Ballerina libraries](https://central.ballerina.io/search?q=org%3Aballerina%2Cballerinax&page=1) are GraalVM-compatible. Therefore packages developed only using these libraries are also GraalVM-compatible. Furthermore, Ballerina reports warnings when the GraalVM build is executed for a project with GraalVM-incompatible packages.
```

File: components/ai/code/ai-bbe/create-images-with-dall-e.md (hunk 7 above)
Comment (praneesha) [reply components/ai/code/ai-bbe/create-images-with-dall-e.md]: Saying `ballerina libraries` is incorrect as per https://github.com/ballerina-platform/ballerina-dev-website/pull/7782#issuecomment-1723704068. Instead, we can have the sentence below.

`The power of the Ballerina library makes uploading, downloading, and processing images a breeze.`

@anupama-pathirage

File: swan-lake/development-tutorials/build-a-graalvm-executable/graalvm-executable-overview.md (hunk 8 above)
Comment (praneesha) [reply swan-lake/development-tutorials/build-a-graalvm-executable/graalvm-executable-overview.md:39]: Saying `ballerina libraries` is incorrect as per https://github.com/ballerina-platform/ballerina-dev-website/pull/7782#issuecomment-1723704068.

File: components/ai/code/ai-bbe/create-images-with-dall-e.md (hunk 7 above)
Comment (anupama-pathirage) [reply components/ai/code/ai-bbe/create-images-with-dall-e.md]: +1

Comment (praneesha) [reply components/ai/code/ai-bbe/create-images-with-dall-e.md]: Updated.

**Review by anupama-pathirage — APPROVED** [review]

**Review by NipunaRanasinghe — APPROVED** [review]

**Pull request merged**